docs/source/deduplication.rst

```diff
@@ -0,0 +1,101 @@
+                    "source_id": 5,
+                    "source_name": "S62Source",
+                    "source_type": "satellite",
+                    "raw_fact_key": null
+                }
+            }
+        ],
+        "entitlements": [
+            {
+                "name": "Satellite Tools 6.3",
+                "entitlement_id": 54,
+                "metadata": {
+                    "source_id": 5,
+                    "source_name": "S62Source",
+                    "source_type": "satellite",
+                    "raw_fact_key": "entitlements"
+                }
+            }
+        ],
+        "metadata": {
+            "os_release": {
+                "source_id": 5,
+                "source_name": "S62Source",
+                "source_type": "satellite",
+                "raw_fact_key": "os_release"
+            }
+        },
+        "sources": [
+            {
+                "id": 5,
+                "source_type": "satellite",
+                "name": "S62Source"
+            }
+        ]
+    }
+
+A scan produces system facts.  As an example, the ``os_release`` describes the operation system and release used by the system.  For each system fact, there is a corresponding entry in the metadata object that identifies the original source of the system fact.
+
+Each fingerprint contains an entitlement list.  The entitlement has a name, id, and metadata describing the original source.  In the example previous example, the system has the ``Satellite Tools 6.3`` entitlement.
+
+A system fingerprint has a list of products.  A product has a name, version, presence, and metadata field.  The system above has JBoss EAP installed.
+
+Last, each system fingerprint has a list of sources which contained this system.  A system may be contained in more than one source.
+
+
+System Deduplication and Merging
+===================
+Quipucords is used to inspect and gather information about your IT infrastructure.  System information can be gathered using the following types of sources:
+
+- network
+- vcenter
+- satellite
+
+A scan is composed of one or more sources. A single system may be found in multiple sources during a scan. For example, a virtual machine on vCenter server may be running a Satellite managed RHEL OS installation. A network source could also be used to scan this system. In this case, the system will be reported via vcenter, satellite, and network sources during a scan. Quipucords feeds unprocessed system facts from a scan into a fingerprint engine. The fingerprint engine matches and merges data for systems seen in more than one source.
+
+Deduplication of Systems
+------------------------
+Quipucords uses specific system facts to identify duplicate systems. The following phases remove duplicate systems:
+
+1. All systems from network sources are combined into a single network system set. Systems are considered to be a duplicate if they have the same fact value for ``subscription_manager_id`` or ``bios_uuid``.
```

Comment:
Systems are considered to be duplicates if they... (all statements in this section)